mule-migration-tool-library/src/main/resources/report.yaml

```diff
@@ -1070,6 +1070,9 @@ raml:
   noMatchingConfig:
     type: ERROR
     message: No matching config was found for RAML element.
+  invalid:
+    type: ERROR
+    message: "Ramlfile invalid: '{syntax}' Please, verify the raml syntax."
```

Comment:
Proposed message: `API specification could not be analyzed: URI parameters variables will not be introduced in corresponding flows.`